storage/database/pebbledb_database.go

```diff
@@ -0,0 +1,318 @@
+package database
+
+import (
+	"bytes"
+	"fmt"
+	"runtime"
+	"sync"
+	"time"
+
+	"github.com/klaytn/klaytn/log"
+
+	"github.com/cockroachdb/pebble"
+	"github.com/cockroachdb/pebble/bloom"
+)
+
+const (
+	minCache = 16
+	minHandles = 16
+	metricsGatheringInterval = 3 * time.Second
+	degradationWarnInterval = time.Minute
+)
+
+type pebbleDB struct {
+	fn string     
+	db *pebble.DB 
+
+	quitLock sync.RWMutex   
+	quitChan chan chan error
+	closed   bool           
+
+	log log.Logger
+
+	writeOptions *pebble.WriteOptions
+}
+
+
+type panicLogger struct{}
+
+func (l panicLogger) Infof(format string, args ...interface{}) {
+}
+func (l panicLogger) Errorf(format string, args ...interface{}) {
+}
+func (l panicLogger) Fatalf(format string, args ...interface{}) {
+	panic(fmt.Errorf("fatal: "+format, args...))
+}
```

Comment:
There's a module variable `logger` in db_manager.go

```suggestion
type panicLogger struct{}

func (l panicLogger) Infof(format string, args ...interface{}) {
	logger.Info(fmt.Sprintf(format, args...))
}
func (l panicLogger) Errorf(format string, args ...interface{}) {
	logger.Error(fmt.Sprintf(format, args...))
}
func (l panicLogger) Fatalf(format string, args ...interface{}) {
	logger.Crit(fmt.Sprintf(format, args...))
}
```

Reply:
Resolved